website/untp-v1.jsonld

```diff
@@ -1,37 +1,86 @@
 {
     "@context": {
         "@version": 1.1,
-        "@vocab": "https://uncefact.github.io/issuer-dependent#",
+        "@vocab": "https://www.w3.org/ns/credentials/issuer-dependent#",
```

Comment:
Why would we point our vocab there? That's not what we're doing here and why people would use our context. 

`https://uncefact.github.io/issuer-dependent#` isn't correct either, it should be something like uncefact.github.io/spec-untp/issuer-dependent#`.

Reply:
The reason I changed for https://www.w3.org/ns/credentials/issuer-dependent is because that page will still return content that will explain to the consumer of the credential what a issuer-dependent term is. Ideally, no terms should get picked up by the `@vocab` and it feels redundant to include a `@vocab` attribute in the context since it's already part of the vcdm2.0 (although that could change). My preference would be to have no `@vocab` in the context. This being said I'm happy to use whichever link you feel is most appropriate here.

Reply:
If we're also just pointing at "https://www.w3.org/ns/credentials/issuer-dependent#", then we could just not have @vocab and let the VC context catch undefined terms.